docsource/installation.rst

```diff
@@ -0,0 +1,208 @@
+    Out[11]:
+    [[1,
+    u'myobj',
+    ...
+    ]]
+
+
+Create a new terminal and start mongo to check to make sure the event has been written. Note that the status is UNPROC (unprocessed event).
+
+.. code-block:: bash
+
+    $ mongo
+    > show dbs
+    Search_test
+    admin
+    local
+    workspace
+    ws_types
+    > use Search_test
+    switched to db Search_test
+    > db.getCollectionNames()
+    ["searchEvents"]
+    > db.searchEvents.findOne()
+    {
+          "_id": ...,
+          "strcde": "WS",
+          "accgrp": 1,
+          ...
+          "status": "UNPROC"
+    }
+
+6. Create a new terminal and edit search_tools.cfg, create a test data type and build the executable script.
+
+.. code-block:: bash
+
+    $ cd [PATH_TO_YOUR_KBaseSearchEngine_DIR]
+    $ git checkout develop
+    $ git pull
+    $ cp search_tools.cfg.example search_tools.cfg
+    $ gedit search_tools.cfg
+
+Make the following edits. Note the user for the token used below must have workspace admin privileges.
+
+.. code-block:: cfg
+
+    search-mongo-host=localhost
+    search-mongo-db=Search_test
+    elastic-host=localhost
+    elastic-port=9200
+    scratch=/home/apasha/dev/kbase/KBaseSearchEngine/temp_scratch
+    workspace-url=http://localhost:7058
+    auth-service-url=https://ci.kbase.us/services/auth/api/legacy/KBase/Sessions/Login
+    indexer-token=[YOUR_CI_TOKEN]
+    types-dir=[PATH_TO_YOUR_KBaseSearchEngine_DIR]/KBaseSearchEngine/test_types
+    type-mappings-dir=[PATH_TO_YOUR_KBaseSearchEngine_DIR]/KBaseSearchEngine/test_type_mappings
+    workspace-mongo-host=fake
+    workspace-mongo-db=fake
+
+.. code-block:: bash
+
+    $ mkdir test_types
+    $ cd test_types
+    $ gedit Empty.json
+
+.. code-block:: json
+
+    {
+        "global-object-type": "EmptyAType2",
+        "ui-type-name": "A Type",
+        "storage-type": "WS",
+        "storage-object-type": "Empty.AType",
+        "indexing-rules": [
+            {
+                "path": "whee",
+                "keyword-type": "string"
+            },
+            {
+                "path": "whee2",
+                "keyword-type": "string"
+            }
+        ]
+    }
+
+.. code-block:: bash
+
+    $ cd ..
+    $ mkdir test_type_mappings
+    $ make build-executable-script JARS_DIR=[ABSOLUTE_PATH_TO_KBASE_JARS_DIR] KB_RUNTIME=[PATH_TO_YOUR_ANT_INSTALL_DIR (example /usr/share)]
+
+8. Start a worker
+
+.. code-block:: bash
+
+    $ bin/search_tools.sh -c search_tools.cfg -k myworker
+    Press return to shut down process
+
+9. Start the coordinator. Note that the event is processed and data has been indexed.
+
+.. code-block:: bash
+
+    $ bin/search_tools.sh -c search_tools.cfg -s
+    Press return to shut down process
+    Moved event xxx NEW_VERSION WS:1/1/1 from UNPROC to READY
+    Moved event xxx NEW_VERSION WS:1/1/1 completed processing with state INDX on myworker
```

Comment:
The last line still doesn't look right - it's like a move line and a completed processing line were smushed together
https://github.com/kbase/KBaseSearchEngine/blob/master/lib/src/kbasesearchengine/main/IndexerCoordinator.java#L274

Reply:
Fixed.